src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,376 @@
+import {ArrowLeftIcon} from '@primer/octicons-react'
+import Link from '../Link'
+import {ForwardRefComponent as PolymorphicForwardRefComponent} from '../utils/polymorphic'
+
+const REGION_ORDER = {
+  ContextArea: 0,
+  TitleArea: 1,
+  Description: 2,
+  Navigation: 3
+}
+
+// Types that are shared between sub components
+export type sharedPropTypes = {
+  visible?: boolean | ResponsiveValue<boolean>
+} & SxProp
+
+// Default state for the `visible` prop when a sub component is only visible on narrow viewport
+const onlyVisibleOnNarrowView = {
+  narrow: true,
+  regular: false,
+  wide: false
+}
+
+// Default state for the `visible` prop when a sub component is visible on regular and wide viewport
+const visibleOnRegularView = {
+  narrow: false,
+  regular: true,
+  wide: true
+}
+
+// Root
+// -----------------------------------------------------------------------------
+export type PageHeaderProps = {
+  'aria-label'?: React.AriaAttributes['aria-label']
+  as?: React.ElementType
+} & sharedPropTypes
+
+const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx = {}}) => {
+  const rootStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+    // 24px for wide and regular, 16px for narrow viewports
+    padding: useResponsiveValue(
+      {
+        narrow: 3,
+        regular: 4,
+        wide: 4
+      },
+      false
+    )
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(rootStyles, sx)}>{children}</Box>
+}
+
+// PageHeader.ContextArea : Only visible on narrow viewports by default to provide user context of where they are at their journey. `visible` prop available
+// to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
+// PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
+// ---------------------------------------------------------------------
+
+const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
+  children,
+  visible = onlyVisibleOnNarrowView,
+  sx = {}
+}) => {
+  const isVisible = useResponsiveValue(visible, false)
+  const contentNavStyles = {
+    display: isVisible ? 'flex' : 'none',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '8px',
+    order: REGION_ORDER.ContextArea
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(contentNavStyles, sx)}>{children}</Box>
+}
+// adopted from React.AnchorHTMLAttributes
+type LinkProps = {
+  download?: string
+  href?: string
+  hrefLang?: string
+  media?: string
+  ping?: string
+  rel?: string
+  target?: string
+  type?: string
+  referrerPolicy?: React.AnchorHTMLAttributes<HTMLAnchorElement>['referrerPolicy']
+}
+export type ParentLinkProps = PageHeaderProps & LinkProps
+
+const ParentLink = React.forwardRef<HTMLAnchorElement, ParentLinkProps>(
+  (
+    {
+      children,
+      sx = {},
+      href,
+      'aria-label': ariaLabel = `Back to ${children}`,
```

Comment:
I would have expected the content (`children`) to be the label of the `<a>`, is there a specific reason for the overriding `aria-label` in this scenario? Is it that we always want to prepend `Back to` before the text?

Reply:
This one is on the [Figma Specs ](https://www.figma.com/file/Ee0OrXuOLXMDqUW83EnDHP/PageHeader-(FY23-Q1)?node-id=2%3A7403&t=a2tLMyUcq6SHAR87-0) but we could definitely discuss its necessity. I was thinking its purpose to give more context ([example](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8)) but open to feedback as always 🙏🏼

Reply:
Curious to hear what a11y folks think about this. Tagging this week's a11y FR @alexislucio